cmd/utils/flaggroup.go

```diff
@@ -0,0 +1,319 @@
+// Copyright 2020 The klaytn Authors
+// This file is part of the klaytn library.
+//
+// The klaytn library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The klaytn library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the klaytn library. If not, see <http://www.gnu.org/licenses/>.
+
+package utils
+
+import (
+	"sort"
+
+	"github.com/klaytn/klaytn/api/debug"
+	"gopkg.in/urfave/cli.v1"
+)
+
+const uncategorized = "MISC" // Uncategorized flags will belong to this group
+
+// FlagGroups categorizes flags into groups to print structured help.
+var FlagGroups = []FlagGroup{
```

Comment:
How about changing this to a map with key as flag name and value as flag group name?

Reply:
Good idea. I added a `TODO` comment about it.